src/screens/Topics/ListTopics.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import {Pressable, StyleSheet, Text} from 'react-native';
 import {COLORS} from '../../utils/theme';
+import {fonts} from '../../utils/fonts';
 
 const ListTopics = ({item, i, myTopic, handleSelectedLanguage}) => (
   <Pressable
     onPress={() => handleSelectedLanguage(item.topic_id)}
     key={i}
     style={[
       styles.bgTopicSelectNotActive,
-      {backgroundColor: myTopic[item.topic_id] ? COLORS.signed_primary : COLORS.concrete}
+      {backgroundColor: myTopic[item.topic_id] ? COLORS.signed_primary : COLORS.gray100}
     ]}>
     <Text>{item.icon}</Text>
     <Text
       style={[
         styles.textTopicNotActive,
-        {color: myTopic[item.topic_id] ? COLORS.white : COLORS.mine_shaft}
+        {color: myTopic[item.topic_id] ? COLORS.white2 : COLORS.white2}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change in the `ListTopics` component seems to be consistent with the PR title. However, there's a redundancy in line 18 where both conditions result in the same color `COLORS.white2`. If this is intentional and you want to keep the condition for future changes, it's fine. Otherwise, you can simplify it.
```diff
- {color: myTopic[item.topic_id] ? COLORS.white2 : COLORS.white2}
+ {color: COLORS.white2}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->